### PR TITLE
NN-3658: Incident details page

### PR DESCRIPTION
**Pull request by natclamp-moj**
Adds the incident details page and connects to the existing statement page, making corrections where necessary

### CODE REVIEW

File: server/data/PrisonLocationResult.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  agencyId: string
+  parentLocationId: number
+  currentOccupancy: number
+  locationPrefix: string
+  operationalCapacity: number
+  userDescription: string
+  internalLocationCode: string
+}
+
+export type PrisonLocation = Location
```
Comment (sp-watson) [inline server/data/PrisonLocationResult.ts:22]: Oh - is this a TS feature I don't know? Why not just define these as types?

File: server/routes/incidentDetails/index.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import express, { RequestHandler, Router } from 'express'
+import asyncMiddleware from '../../middleware/asyncMiddleware'
+
+import IncidentDetailsRoutes from './incidentDetails'
+
+import PlaceOnReportService from '../../services/placeOnReportService'
+import LocationService from '../../services/locationService'
+
+export default function prisonerIncidentStatementsRoutes({
```
Comment (sp-watson) [inline server/routes/incidentDetails/index.ts:9]: Minor naming issue - details not statement

**Review by sp-watson — DISMISSED** [review]

File: server/routes/incidentDetails/index.ts (hunk 2 above)
Comment (natclamp-moj) [reply server/routes/incidentDetails/index.ts:9]: good catch!

File: server/data/PrisonLocationResult.ts (hunk 1 above)
Comment (natclamp-moj) [reply server/data/PrisonLocationResult.ts:22]: Good point, something that came with merging a couple of files but i'll simplify

**Review by sp-watson — APPROVED** [review]

**Pull request merged**